src/Process/Runner/Parallel/ParallelProcessRunner.php

```diff
@@ -52,14 +52,6 @@
  */
 class ParallelProcessRunner
```

Comment:
Those changes seem unrelated no? Would it be possible it to extract it in a dedicated PR to evaluate its impact there?

Reply:
Yes, let's try this.